app/translations/src/TranslationFragment.js

```diff
@@ -25,12 +25,14 @@ export default class TranslationFragment extends React.Component<{|
     if (this.props.children === undefined) {
       throw new Error("'TranslationFragment' cannot be used without children.");
     }
-
-    return React.Children.map(this.props.children, child =>
-      React.cloneElement(child, {
-        ...this.props,
-        children: null,
-      }),
-    );
+    return React.Children.map(this.props.children, child => {
+      if (child) {
+        return React.cloneElement(child, {
```

Comment:
This change is necessary because condintional useage like 
```
<TranslationFragment>
        {stars.includes(0) && (
          <TranslationFragment>
            <Translation
              id="HotelsSearch.Filter.StarsFilter.Unrated"
              key="unratedTranslation"
            />
            {stars.length > 1 && <DummyTranslation id="," />}
          </TranslationFragment>
        )}
        <DummyTranslation
          id={stars
            .sort()
            .filter(star => star > 0)
            .join(',')}
        />
      </TranslationFragment>
```
gives false as a child, and React.cloneElement(false) crashes.